### DISCUSSION TITLE
Auto-complete/List View default query.

### DISCUSSION THREAD

**Discussion by ssimo3lsuhsc** [pods-framework/pods#6778] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to type something in an auto-complete or List View field because I can't remember the titles of the posts I added recently. The post I want to add to the field value array is likely the one I just created.

**Describe the solution you'd like**
Instead of displaying placeholder text, "Please enter 1 or more characters," auto-complete and list view fields should run a default query. The default query will display the 5 most recent items that satisfy the criteria for eligibility for selection.

**Describe alternatives you've considered**
I've considered adding a bulk menu option to the Post Edit list screen. In testing, this does not work for Pods Settings pages: I can get the option, unshift a value to it, and then request that it be re-saved. However, it does not seem that Pods knows about the change. I've even tried to use `wp_cache_delete('alloptions','options')` before updating the options from the Post Edit list screen. I've also tried Carbon Fields, but it would be nice to have this option with Pods because I use Pods to manage some custom post types.

**Comment by quasel** [comment]
intresting idea to speed things up @pglewis what's your take on it?
